files/active_record_relation_patch.rb

```diff
@@ -0,0 +1,7 @@
+module ActiveRecord
+  class Relation
+    def to_s
+      "Array of #{self.count} #{self.model.to_s.pluralize}"
```

Comment:
@jelaniwoods  Oh — actually, this is a nice touch; but I didn't want this pluralized on purpose. I want it to be painfully obvious that these are instances of the model class.

Reply:
@raghubetina Gotcha, that makes sense.